Host a Static Website with Hugo and NGINX.md

```diff
@@ -45,7 +45,7 @@ Throughout this guide commands will be given with example variables to provide g
 | username                 | Replace with Linode login user name                   |
 
 
-Begin by completing the Getting Started guide and Securing Your Server guide if you are getting started with Linode for the first time.
+Begin by completing the following guides if you are getting started with Linode for the first time.
```

Comment:
Listing the guides ahead of time without links, makes for a less than ideal user experience. So just tell 'em it's coming.

Reply:
This guide is linked to Linode's internal docs.

Reply:
Yes, but no need to list them twice. Just say the links are below.